libs/community/langchain_community/vectorstores/tidb_vector.py

```diff
@@ -0,0 +1,336 @@
+import uuid
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple
+
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+from langchain_core.vectorstores import VectorStore
+
+DEFAULT_DISTANCE_STRATEGY = "cosine"  # or "l2"
+DEFAULT_TiDB_VECTOR_STORE_NAME = "langchain_tidb_vector"
+
+
+class TiDBVector(VectorStore):
+    def __init__(
+        self,
+        connection_string: str,
+        embedding_function: Embeddings,
+        tidb_vectorstore: Optional[Any] = None,
+        vectorstore_name: str = DEFAULT_TiDB_VECTOR_STORE_NAME,
```

Comment:
Any reason why we dont want to call it table_name ? thats the convention when it comes to MySQL. 

By providing a default DEFAULT_TiDB_VECTOR_STORE_NAME are we saying that if a user doesn't provide a table name then a default table will be created ? If so what would be its structure ?

Reply:
addressed